packages/@aws-cdk/aws-events-targets/lib/log-group-resource-policy.ts

```diff
@@ -0,0 +1,54 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as cdk from '@aws-cdk/core';
+import * as cr from '@aws-cdk/custom-resources';
+
+/**
+ * Properties to configure a log group resource policy
+ */
+export interface LogGroupResourcePolicyProps {
+  /**
+   * The log group resource policy name
+   */
+  readonly policyName: string;
+  /**
+   * The policy statements for the log group resource logs
+   */
+  readonly policyStatements: [iam.PolicyStatement];
+}
+
+/**
+ * Creates LogGroup resource policies.
+ */
+export class LogGroupResourcePolicy extends cr.AwsCustomResource {
+  constructor(scope: cdk.Construct, id: string, props: LogGroupResourcePolicyProps) {
+    const policyDocument = new iam.PolicyDocument({
+      statements: props.policyStatements,
+    });
+
+    super(scope, id, {
+      resourceType: 'Custom::CloudwatchLogResourcePolicy',
+      onUpdate: {
+        service: 'CloudWatchLogs',
+        action: 'putResourcePolicy',
+        parameters: {
+          policyName: props.policyName,
+          policyDocument: JSON.stringify(policyDocument),
+        },
+        physicalResourceId: cr.PhysicalResourceId.of(id),
+      },
+      onDelete: {
+        service: 'CloudWatchLogs',
+        action: 'deleteResourcePolicy',
+        parameters: {
+          policyName: props.policyName,
+        },
+        ignoreErrorCodesMatching: '400',
+      },
+      policy: cr.AwsCustomResourcePolicy.fromSdkCalls({
+        // putResourcePolicy and deleteResourcePolicy don't support resource-level permissions. We must specify all resources ("*").
+        // https://docs.aws.amazon.com/IAM/latest/UserGuide/list_amazoncloudwatchlogs.html
+        resources: ['*'],
```

Comment:
I think we could use `AwsCustomResourcePolicy.ANY_RESOURCE` here instead of `*`. This way, this piece of code would be aligned with `AwsCustomResources` readme.

https://docs.aws.amazon.com/cdk/api/latest/docs/custom-resources-readme.html#execution-policy-1